mcs/class/corlib/System/Enum.cs

```diff
@@ -392,7 +392,7 @@ public static string[] GetNames (Type enumType)
 			MonoEnumInfo.GetInfo (enumType, out info);
 			return (string []) info.names.Clone ();
 		}
-
+/*
```

Comment:
Okay, I know see this is just for enums which probably don't have too many items to search for. So the boxing is not great, but probably not a real issue.